src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString().toLowerCase();
```

Comment:
Same as from the previous assignment. Previous comments:

```
Do we really need to create a new instance of Random every time when this method is called?
It is better to create a new Random field as class property.
```

```
Your index has the same value on each getRandomColor() call. 
Let's better calculate index value within the method and make your Random instance a class field (initialized directly in the field or using a constructor)
```

Also, [Use name() for getting String representation of enum constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants).